Bio/Seq.py

```diff
@@ -468,29 +397,42 @@ def __ge__(self, other):
             return self._data >= other
 
     def __len__(self):
-        """Return the length of the sequence, use len(my_seq)."""
+        """Return the length of the sequence."""
         return len(self._data)
 
     def __getitem__(self, index):
-        """Return a subsequence of single letter, use my_seq[index].
+        """Return a subsequence as a single letter or as a sequence object.
 
-        >>> my_seq = Seq('ACTCGACGTCG')
-        >>> my_seq[5]
+        If the index is an integer, a single letter is returned as a Python
+        string:
+
+        >>> seq = Seq('ACTCGACGTCG')
+        >>> seq[5]
         'A'
+
+        Otherwise, a new sequence object of the same class is returned:
+
+        >>> seq[5:8]
+        Seq('ACG')
+        >>> mutable_seq = MutableSeq('ACTCGACGTCG')
+        >>> mutable_seq[5:8]
+        MutableSeq('ACG')
         """
         if isinstance(index, int):
             # Return a single letter as a string
             return chr(self._data[index])
         else:
-            # Return the (sub)sequence as another Seq object
+            # Return the (sub)sequence as another Seq/MutableSeq object
             return self.__class__(self._data[index])
 
     def __add__(self, other):
-        """Add another sequence or string to this sequence.
+        """Add a sequence or string to this sequence.
 
-        >>> from Bio.Seq import Seq
+        >>> from Bio.Seq import Seq, MutableSeq
         >>> Seq("MELKI") + "LV"
         Seq('MELKILV')
+        >>> MutableSeq("MELKI") + "LV"
+        MutableSeq('MELKILV')
         """
         if isinstance(other, (Seq, MutableSeq)):
```

Comment:
Should this be checking for instances of the new ABC?

Reply:
(First example of many)

Reply:
Fixed